pvr.iptvsimple/resources/instance-settings.xml

```diff
@@ -106,7 +106,36 @@
         </setting>
       </group>
 
-      <group id="3" label="30006">
+      <group id="3" label="30078">
+        <setting id="connectioncheckinterval" type="integer" label="30080" help="30629">
+          <level>1</level>
+          <default>10</default>
+          <constraints>
+            <minimum>1</minimum>
+            <step>1</step>
+            <maximum>60</maximum>
+          </constraints>
+          <control type="slider" format="integer">
+            <popup>true</popup>
+            <formatlabel>14045</formatlabel>
+          </control>
+        </setting>
+        <setting id="connectionchecktimeout" type="integer" label="30079" help="30628">
+          <level>2</level>
+          <default>20</default>
+          <constraints>
+            <minimum>10</minimum>
```

Comment:
can we please allow this to go down to 1 ?

Reply:
also, the m3uRefreshIntervalMins allows going to 0 which i believe may be a mistake? 
That just makes it constantly refresh.
Maybe refresh interval should have a minimum of 1 or even 10

Reply:
So this is essentially a TCP timeout. In the majority of cases it will just return immediately as you will have seen. The minimum is 10 so any TCP stuff can finish elegantly, lower values than that can be messy generally.

Reply:
Point taken on the refresh, that would be once a second which is far too often. Will change the minimum there to 1.

Reply:
ive found with a local server, if its just a second too early, itll only timeout after 10s (if port not available)
For a local server - 1s is huge amount.
Is there any harm letting it go lower?

Reply:
True, I guess there is no harm. Try and let me know how it  works.